x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/common/monitor_test_result/status_badge.tsx

```diff
@@ -47,7 +47,7 @@ export const getBadgeColorForMonitorStatus = (status: MonitorStatus): IconColor
 export const getTextColorForMonitorStatus = (
   status: MonitorStatus
 ): keyof EuiThemeComputed['colors'] => {
-  return status === 'skipped' ? 'disabledText' : 'text';
+  return status === 'skipped' ? 'textDisabled' : 'text';
```

Comment:
should it be `textParagraph` when not skipped?

Reply:
Nice catch!